scripts/version

```diff
@@ -1,35 +1,36 @@
 #!/bin/bash
 
+DIRTY=
 if [ -n "$(git status --porcelain --untracked-files=no)" ]; then
     DIRTY="-dirty"
 fi
 
-COMMIT=$(git rev-parse --short HEAD)
-GIT_TAG=${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}
+COMMIT=${COMMIT:-$(git rev-parse --short HEAD)}
+GIT_TAG=${GIT_TAG:-${DRONE_TAG:-$(git tag -l --contains HEAD | head -n 1)}}
```

Comment:
Might also be good to remove the `DRONE_TAG` part since we are off drone now.

Reply:
Sure.